docs/lib/examples/import-basic.js

```diff
@@ -1,27 +1,23 @@
 import React from 'react'
 import {Player} from 'cueplayer-react'
 import {PlayerProvider} from 'cueplayer-react/context/player-context'
-import SidePanel from '../Components/side-panel/SidePanel'
```

Comment:
this sidepanel was just an example implementation and not part of the library

Reply:
it's demonstrating the capability with the provider

Reply:
put SidePanel stuff back